### DISCUSSION TITLE
post/meituan-rider

### DISCUSSION THREAD

**Discussion by giscus** [huanfe1/blog#26] [Announcements]
# post/meituan-rider

暑假从事了两个月的美团骑手，这是我第一份长时间的全职工作，所以觉得还是有必要记录下的，顺便也给没接触过这份职业...

https://www.huanfei.top/post/meituan-rider

**Comment by istarry123** [comment]
牛的

**Comment by aragakey** [comment]
写得真好

**Comment by b-yp** [comment]
认真生活的人
